frontend/src/MainBlock/Artifacts/Artifacts.js

```diff
@@ -0,0 +1,62 @@
+import React, { Fragment, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import List from '@material-ui/core/List';
+import Divider from '@material-ui/core/Divider';
+import { withStyles } from '@material-ui/core/styles';
+
+import { fetchArtifactTree } from 'redux/models/Artifact/artifactsActions';
+import ArtifactRepositoryItem from './ArtifactRepositoryItem';
+
+const styles = theme => ({
+    root: {
+        padding: theme.spacing.unit * 2,
+        width: theme.spacing.unit * 100,
+    },
+});
+
+function Artifacts({ classes, artifactTree, fetchArtifactTree }) {
+    useEffect(() => {
+        fetchArtifactTree();
+    }, []);
+
+    if (typeof artifactTree !== 'undefined' && Array.isArray(artifactTree)) {
```

Comment:
```suggestion
    if (artifactTree) {
```
То, что это массив определяется PropTypes